src/main/java/org/profit/telegrambot/TgBot.java

```
@@ -0,0 +1,208 @@
+                newMessage.setText(answer);
+                addCounter ++;
+
+                addToCart();
+                try {
+                    execute(newMessage);
+                } catch (TelegramApiException e) {
+                    e.printStackTrace();
+                }
+            }
+        } else if (update.getMessage().hasPhoto()){
+
+            List<PhotoSize> photos = update.getMessage().getPhoto();
+            String fileId = Objects.requireNonNull(photos.stream()
+                    .sorted(Comparator.comparing(PhotoSize::getFileSize).reversed())
+                    .findFirst()
+                    .orElse(null)).getFileId();
+
+            System.out.println(fileId);
+        } // This is a method to get photo's fileId to use it later
+    }
+
+    /*public synchronized void sendMessage(String chatId, String text){
+        SendMessage message = new SendMessage();
+        message.setChatId(chatId);
+        message.setText(text);
+
+        try{
+            execute(message);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
+        }
+    }*/ // Currently in doubt whether this method is needed or not
+    public synchronized void sendMessage(String chatId, String text, ReplyKeyboardMarkup markup){
+        SendMessage message = new SendMessage();
+        message.setChatId(chatId);
+        message.setText(text);
+        message.setReplyMarkup(markup);
+        try{
+            execute(message);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
+        }
+    }
+    public synchronized void sendMessage(String chatId, String text, InlineKeyboardMarkup markup){
+        SendMessage message = new SendMessage();
+        message.setChatId(chatId);
+        message.setText(text);
+        message.setReplyMarkup(markup);
+        try{
+            execute(message);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
```

Like for not ignoring exceptions and letting them be logged.